jsonnet/components/console-plugin.libsonnet

```diff
@@ -0,0 +1,16 @@
+local cm = (import 'console-plugin/config-map.json');
+local deploy = (import 'console-plugin/deployment.json');
+local cp = (import 'console-plugin/console-plugin.json');
+local pdb = (import 'console-plugin/pod-disruption-budget.json');
+local sa = (import 'console-plugin/service-account.json');
+local svc = (import 'console-plugin/service.json');
```

Comment:
These files are generated just like manifests are to `tmp/<random/assets` (instead of `tmp/random/manifests`) and included by the `build-jsonnet` script.

Reply:
So the way this works is that  `make build-jsonnet` will first convert `assets/console-plugin/*.yaml` to `tmp/<rand>/console-plugin/*.json` and then this directory is included in the jsonnet path (-J) by build-jsonnet.sh script. These files are imported by the `console-plugin.libsonnet` which is then used to generate the yaml files. So essentially, we just need to maintain the yaml files until this is completely replaced by proper jsonnet in a follow up PR.